### PR TITLE
Migrate asset and authors helpers to es6

### PR DESCRIPTION
**Pull request by vikaspotluri123**
Refs #9589 

- Update code style for asset helper
- Migrate authors helper

### CODE REVIEW

File: core/server/helpers/authors.js [hunk 1]
```diff
@@ -6,27 +6,31 @@
...
+const {SafeString, templates, models} = proxy;
 
-    const autolink = !(_.isString(options.hash.autolink) && options.hash.autolink === 'false'),
-        separator = _.isString(options.hash.separator) ? options.hash.separator : ', ',
-        prefix = _.isString(options.hash.prefix) ? options.hash.prefix : '',
-        suffix = _.isString(options.hash.suffix) ? options.hash.suffix : '',
-        limit = options.hash.limit ? parseInt(options.hash.limit, 10) : undefined,
-        visibilityArr = models.Base.Model.parseVisibilityString(options.hash.visibility);
+module.exports = function authors(options = {}) {
+    options.hash = options.hash || {};
```
Comment (vikaspotluri123) [inline core/server/helpers/authors.js]: Didn't use nested destructuring here because the input can be considered untrusted

**Comment by kirrg001** [comment]
@rishabhgrg Could you please review & merge next Monday? Thaaanks 🙂

File: core/server/helpers/authors.js [hunk 2]
```diff
@@ -6,27 +6,31 @@
...
-module.exports = function authors(options) {
-    options = options || {};
-    options.hash = options.hash || {};
+const proxy = require('./proxy');
+const _ = require('lodash');
+const urlService = require('../services/url');
+const {SafeString, templates, models} = proxy;
 
-    const autolink = !(_.isString(options.hash.autolink) && options.hash.autolink === 'false'),
-        separator = _.isString(options.hash.separator) ? options.hash.separator : ', ',
```
[the remark this answers is not in the document]
Comment (rishabhgrg) [reply core/server/helpers/authors.js]: 🤔Thinking if losing the `isString` check in the destructuring default value [here](https://github.com/TryGhost/Ghost/pull/10611/commits/c8dcaa2090002e05922ae36588352f03b6431f9e#diff-769c52dea22d98606ffae06e4266a5b7R19) has any downsides. Seems it differs only when non-string value is passed to `separator`, which should be wrong usage so can be ignored as case.

[the remark this answers is not in the document]
Comment (vikaspotluri123) [reply core/server/helpers/authors.js]: Yeah, the only possible case I can think of is if the theme author passes in a variable (e.g. number of posts) 😟

[the remark this answers is not in the document]
Comment (rishabhgrg) [reply core/server/helpers/authors.js]: @vikaspotluri123 Ah yeah, but passing number of posts in as a separator is still wrong usage in a way right? 😄I mean previously we would have just ignored it and set `, ` as separator, so I think the changes are still ok ;) Wdyt?

[the remark this answers is not in the document]
Comment (vikaspotluri123) [reply core/server/helpers/authors.js]: I agree 😄

**Pull request merged**